content/docs/started/getting-started.md

```diff
@@ -1,109 +1,177 @@
 +++
 title = "Getting Started with Kubeflow"
-description = "Quickly get running with your ML Workflow on an existing Kubernetes installation"
+description = "Quickly get running with your ML Workflow"
 weight = 1
 +++
 
 ## Who should consider using Kubeflow?
 
 Based on the current functionality you should consider using Kubeflow if:
 
-  * You want to train/serve TensorFlow models in different environments (e.g. local, on prem, and cloud)
+  * You want to train/serve TensorFlow models in different environments (e.g.
+    local, on prem, and cloud)
   * You want to use Jupyter notebooks to manage TensorFlow training jobs
   * You want to launch training jobs that use resources -- such as additional
     CPUs or GPUs -- that aren't available on your personal computer
   * You want to combine TensorFlow with other processes
-       * For example, you may want to use [tensorflow/agents](https://github.com/tensorflow/agents) to run simulations to generate data for training reinforcement learning models.
+       * For example, you may want to use
+	 [tensorflow/agents](https://github.com/tensorflow/agents) to run
+	 simulations to generate data for training reinforcement learning
+	 models.
+
+This list is based ONLY on current capabilities. We are investing significant
+resources to expand the functionality and actively soliciting help from
+companies and individuals interested in contributing (see
+[Contributing](/docs/contributing/)).
+
+## Installation without pre-existing Kubernetes
+
+Here is how you can get Kubeflow up and running if you don't have a K8s cluster
+running already.
+
+### Local
+
+There are a several options:
+
+   * [MiniKF](/docs/started/getting-started-minikf/)
```

Comment:
For consistency with the other bullet points in the list:

s/MiniKF/MiniKF setup

Reply:
yes, you are right, thanks